AdyenCashAppPay/CashAppPayComponent.swift

```diff
@@ -181,7 +163,25 @@ public final class CashAppPayComponent: PaymentComponent,
                                  customerId: customerId,
                                  cashtag: cashtag)
     }
+
+    internal func createPaymentActions() -> [PaymentAction] {
+        var actions = [PaymentAction]()
+        if let amount = payment?.amount, amount.value > 0 {
+            let moneyAmount = Money(amount: UInt(amount.value), currency: .USD)
+            let oneTimeAction = PaymentAction.oneTimePayment(scopeID: cashAppPayPaymentMethod.scopeId,
+                                                             money: moneyAmount)
+            actions.append(oneTimeAction)
+        }
     
+        if storePayment == true {
```

Comment:
No need to check for `true` anymore as it's not optional anymore 👍 
```suggestion
        if storePayment {
```